packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -12,491 +12,674 @@
+        <li>secondary</li>
       </ol>
-      <p>
-        To indicate that a secondary button style should be used for the "text" toggle, add
-        <span class="dummy-code">@color="secondary"</span>. If no
-        <span class="dummy-code">@color</span>
-        is declared,
-        <span class="dummy-code">primary</span>
-        will be used by default.
-      </p>
+    </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
     </dd>
   </dl>
-  <hr />
-  <p class="dummy-paragraph" id="toggle-icon-api">
-    This is the API for the
-    <span class="dummy-code">Dropdown::Toggle::Icon</span>
-    component:
-  </p>
-  <dl class="dummy-component-props" aria-labelledby="toggle-icon-api">
+
+  <h4 class="dummy-h4">Toggle::Icon</h4>
+  <p class="dummy-paragraph" id="component-api-toggle-icon">Here is the API for the icon-only toggle component (yielded
+    as
+    <code class="dummy-code">ToggleIcon</code>):</p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-toggle-icon">
     <dt>text <code>string</code></dt>
     <dd>
       <p>The value of <em>aria-label</em> for the toggle icon.</p>
       <p><em>If no text value is defined an error will be thrown.</em></p>
     </dd>
     <dt>icon <code>string</code></dt>
     <dd>
-      <p>Any valid flight-icon name. If none is defined,
-        <span class="dummy-code default">user</span>
-        is set by default</p>
+      <p>Acceptable value: any Flight icon name.</p>
     </dd>
     <dt>hasChevron <code>boolean</code></dt>
     <dd>
-      <p>
-        Acceptable values:
-      </p>
-      <ol>
-        <li class="default">true</li>
-        <li>false</li>
-      </ol>
+      <p>Default: <span class="default">true</span></p>
       <p>Per design,
-        <span class="dummy-code">false</span>
+        <code class="dummy-code">false</code>
         is only acceptable when the "more-horizontal" icon is used; as such, it is set to
-        <span class="dummy-code">true</span>
+        <code class="dummy-code">true</code>
         by default.</p>
     </dd>
-    <dt>
-      imageSrc
-      <code>string</code>
-    </dt>
+    <dt>imageSrc <code>string</code></dt>
     <dd>
       The URL of an image to be used instead of an icon (e.g., as an avatar).
     </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
   </dl>
-  <hr />
-  <p class="dummy-paragraph" id="listitem-api">
-    This is the API for the
-    <span class="dummy-code">Dropdown::ListItem</span>
-    component:
-  </p>
-  <dl class="dummy-component-props" aria-labelledby="listitem-api">
-    <dt>
-      item
-      <code>
-        enum
-      </code>
-    </dt>
+
+  <h4 class="dummy-h4">ListItem::CopyItem</h4>
+  {{! TODO: add props here if we decide to use the current version }}
+  <p class="dummy-paragraph">[WIP]</p>
```

Comment:
Shouldn't this be removed altogether since it will eventually be its own component and not live within the dropdown?

Reply:
We don't know yet, it may be that to facilitate the adoption of the `Dropdown` we will have a component, and we'll remove it once we have the proper one. Or we just re-use the one from Structrure/Cloud UI inside a `generic` element. The options will be discussed today with @MelSumner